bitnami/redis-cluster/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis-cluster
-version: 2.0.1
+version: 2.0.4
```

Comment:
I know you created another PR, but please don't skip versions. Wait for one of your PRs to be landed, so you can rebase and bump the version again.

Reply:
Sounds good, thanks!